Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -195,7 +212,7 @@ extension PhotonActionSheetProtocol {
             topActions.append(addReadingList)
         }
         
-        return [topActions, [copyURL, findInPageAction, toggleDesktopSite, pinToTopSites], [share]]
+        return [topActions, [copyURL, findInPageAction, toggleDesktopSite, pinToTopSites, sendToDevice], [share]]
```

Comment:
This feature has a new string so it wont be able to land in 10. We'll need to add it behind a feature flag. This should be pretty simple. 
We create a new entry in `AppConstants.swift` and set the feature disabled in beta/release and then here you can just do 
```
        var middleActions = [copyURL, findInPageAction, toggleDesktopSite, pinToTopSites]
        if MOZ_SENDTAB_IN_PAGE_ACTIONS {
            middleActions.append(sendToDevice)
        }
        
        return [topActions, middleActions, [share]]
```

Reply:
We might be able to rework a few strings to have this land in 10. I'll have to talk to Stefan about this.